detection/persistence/unexpected-listening-port-macos.sql

```diff
@@ -178,5 +178,12 @@ WHERE
     AND lp.port > 1024
     AND lp.protocol = 6
   )
+  AND NOT (
+    (
+      p.path LIKE '/System/Library/CoreServices/RemoteManagement/ARDAgent.app/Contents/MacOS/ARDAgent'
```

Comment:
This should be `p.path = ` instead of `p.path LIKE` as the string includes no wildcards. It requires less CPU overhead.

Reply:
Oops!  Thanks for catching that, fixed.  I do wonder though -- if there are no '%' in the string, does it execute like '='?